java/identify-layers/README.md

```diff
@@ -1,21 +1,34 @@
-# Identify Layers
-Identify features on a map across different layers.
+# Identify layers
 
-![Identify Layers App](identify-layers.png)
+Identify features in all layers in a map. 
+
+![Image of identify layers](identify-layers.png)
+
+## Use case
+
+IdentifyLayers allows users to tap on a map, returning features at that location across multiple layers. Because some layer types have sublayers, the sample recursively counts results for sublayers within each layer.
 
 ## How to use the sample
-Tap on map to get features at that location. The features are returned from different layers in the map.
+
+Tap to identify features. An alert will show all layers with features under the cursor, as well as a layer count.
```

Comment:
Sample does not display a layer count, but does display the number of features within each layer.